extension/src/setup/index.ts

```diff
@@ -244,8 +244,12 @@ export class Setup
     return !!this.webview?.isActive
   }
 
-  public shouldBeShown() {
-    return !this.getCliCompatible() || !this.hasRoots() || !this.getHasData()
```

Comment:
[Q] Why not return an enum from this? so that callers don't need to remember to call `isDvcSetup` and then `isExperimentsSetup` in that order.

Reply:
Good idea!